src/Tests/Microsoft.NET.Publish.Tests/GivenThatWeWantToPublishAHelloWorldProject.cs

```diff
@@ -501,8 +501,11 @@ public void It_fails_if_nobuild_was_requested_but_build_was_invoked()
                 .HaveStdOutContaining("NETSDK1085");
         }
 
-        [Fact]
-        public void It_contains_no_duplicates_in_resolved_publish_assets()
+        [Theory]
+        [InlineData("windows")]
```

Comment:
The "windows" test needs to be under a `WindowsOnlyFact`

Reply:
I put a sloppy, if + return in this: https://github.com/dotnet/sdk/compare/f0a30eecf14d6b93e656aed045a907525e5342ee..925903a4c3ff5a1ad2651d2f784ac062e63d9e04

I can refactor to a WindowsOnlyFact.

Reply:
Either way is fine with me.

Reply:
I went ahead and cleaned it up in aaa271b